test/requirements/python-requirements.spec.js

```diff
@@ -0,0 +1,28 @@
+/*jshint node:true, mocha:true*/
+
+'use strict';
+const { spawnSync } = require('child_process');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+describe('python-test', function () {
```

Comment:
No need to add the `-test` suffix (that part is obvious ;) )

Reply:
Ok.